src/core/navigation/navigation.js

```diff
@@ -0,0 +1,44 @@
+import Command from 'core/infrastructure/command';
+
+export default class Navigation {
+	constructor(model) {
+		this.model = model;
+	}
+
+	get commands() {
+		const model = this.model,
+			commands = {
+				goDown: new Command({
+					shortcut: 'down',
+					canExecute: () => model().row < model().rowCount,
+					execute: () => {
+						model({row: model().row + 1});
+					}
+				}),
+				goUp: new Command({
+					shortcut: 'shift+tab|up',
```

Comment:
shift + tab is opposite to tab. So  i think shift + tab should be in goLeft command